### DISCUSSION TITLE
Uploading snapshots with no events through web-ui

### DISCUSSION THREAD

**Discussion by bsacharski** [blakeblackshear/frigate#9043] [Ask A Question] [answered]
Hello

Sorry if the question has been asked before, but is it possible to send a snapshot from an arbitrary moment in any recording to the Frigate+ from web-ui?

The use case is as follows:

As a user I go through my recordings:
1. I see something in a recording that should be recognized as an event (but that wasn't detected)
2. I pause the video
3. I upload to frigate+ the snapshot with the frame that the recording been paused on
4. I go to frigate+ and add annotations for the objects in the snapshot
5. Once I have enough of samples, I re-train the model

Is something like that possible, or maybe planned?

**Comment by NickM-27** [comment, marked as answer]
the frigate+ webUI supports uploading images for cameras, do this in the images page on the top right. For now you have to watching recordings and use right click to save frame (this is a new feature on chrome).

In a future version of frigate this will be supported directly in frigate
